### PR TITLE
feat(i18n): update zh-CN translations

### PR DESCRIPTION
**Pull request by webfansplz**
### Additional context

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Other

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [ ] Provide related snapshots or videos.
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### CODE REVIEW

File: locales/zh-CN.json [hunk 1]
```diff
@@ -338,6 +338,14 @@
...
       "import": "导入用户令牌",
       "label": "当前用户"
+    },
+    "wellness": {
+      "feature": {
+        "hide_boost_count": "隐藏转发数",
+        "hide_favorite_count": "隐藏收藏数",
+        "hide_follower_count": "隐藏关注者数"
+      },
+      "label": "健康设置"
```
Comment (sxzz) [inline locales/zh-CN.json:348]: 这个翻译听起来怪怪的，像是儿童保护什么的……

Comment (webfansplz) [reply locales/zh-CN.json:348]: 😂,我原来还在身心和健康徘徊,然后看了下zh-TW的翻译,所以做了对齐.有什么建议吗?

Comment (antfu) [reply locales/zh-CN.json:348]: The term "Wellness" itself actually makes me feel unwellness as I turn on the like counter 🤣

Comment (antfu) [reply locales/zh-CN.json:348]: 可能 `社交偏好`?

Comment (webfansplz) [reply locales/zh-CN.json]: > The term "Wellness" itself actually makes me feel unwellness as I turn on the like counter 🤣

😂.How about move `Wellness` to the `Preferences` ?

**Pull request closed without merging**